docs/ops/deployment/local.zh.md

```diff
@@ -122,38 +121,38 @@ INFO ... - Recovering all persisted jobs.
 INFO ... - Registering TaskManager ... at ResourceManager
 {% endhighlight %}
 
-#### Windows Cygwin Users
+#### Windows Cygwin 用户
 
-If you are installing Flink from the git repository and you are using the Windows git shell, Cygwin can produce a failure similar to this one:
+如果你使用 Windows 的 git shell 从 git 仓库安装 Flink，Cygwin 可能会产生类似如下的错误：
 
 {% highlight bash %}
 c:/flink/bin/start-cluster.sh: line 30: $'\r': command not found
 {% endhighlight %}
 
-This error occurs because git is automatically transforming UNIX line endings to Windows style line endings when running in Windows. The problem is that Cygwin can only deal with UNIX style line endings. The solution is to adjust the Cygwin settings to deal with the correct line endings by following these three steps:
+这个错误是因为当 git 运行在 Windows 上时，它会自动地将 UNIX 的行结束符转换成 Windows 风格的行结束符，但是 Cygwin 只能处理 Unix 风格的行结束符。解决方案是通过以下三步调整 Cygwin 的配置，使其能够正确处理行结束符：
 
-1. Start a Cygwin shell.
+1. 启动 Cygwin shell。
 
-2. Determine your home directory by entering
+2. 输入以下命令进入你的家目录
 
 {% highlight bash %}
 cd; pwd
 {% endhighlight %}
 
-    This will return a path under the Cygwin root path.
+    这个命令会在 Cygwin 的根目录下返回路径地址。
 
-3. Using NotePad, WordPad or a different text editor open the file `.bash_profile` in the home directory and append the following: (If the file does not exist you will have to create it)
+3. 使用 NotePad、WordPad 或其他的文本编辑器在家目录下打开 `.bash_profile` 文件并添加以下内容：（如果这个文件不存在则需要创建）
 
 {% highlight bash %}
 export SHELLOPTS
 set -o igncr
 {% endhighlight %}
 
-Save the file and open a new bash shell.
+保存文件并打开一个新的 bash shell。
 
-### Stop a Local Flink Cluster
+### 关闭 Flink 本地集群
```

Comment:
Thanks for your explanation. I have added `<a>` tags for all titles in this page.
I wonder if we should always add `<a>` tags for every title? I noticed that some chinese pages' titles don't have their corresponding `<a>` tags so that the links refering to them are invalid, and I have created a [PR](https://github.com/apache/flink/pull/13308) to fix them for one page.
If `<a>` tags are always needed, I will improve that [PR](https://github.com/apache/flink/pull/13308) by adding `<a>` tags for all titles.

Reply:
Personally I prefer to add `<a>` tags for all titles.